Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM python:3.11-slim as base
+
+# install poetry 
+RUN pip install poetry 
+ 
+WORKDIR /opt/todoapp
+COPY . /opt/todoapp
+
+# Configure for development
+
+FROM base as development
+#install dependencies 
+RUN poetry install
```

Comment:
Given that you run this `poetry install` command on both stages, would it be better placed in the shared  `base` stage, e.g. line 10?